quickwit-indexing/failpoints/mod.rs

```diff
@@ -258,7 +260,10 @@ async fn test_merge_executor_controlled_directory_kill_switch() -> anyhow::Resul
 
     let metastore = test_index_builder.metastore();
     let split_infos = metastore.list_all_splits(index_id).await?;
```

Comment:
`split_infos` -> `splits`
`splits` -> `split_metadatas`

Reply:
Let's do the rename somewhere else.